src/contexts/CountContext.jsx

```diff
@@ -0,0 +1,28 @@
+/*import { createContext, useContext, useState, useEffect } from "react";
+
+const CountContext = createContext();
+
+export const CountProvider = ({ children }) => {
+    const [count, setCount] = useState(0);
+
+    function increment() {
+        setCount(function (prevCount) {
+            return (prevcount += 1);
+
+        })
+    }
+}
```

Comment:
it's okay to declare a function this way, but keep it consistent and if you used `const = () => {}`syntax  before, stick to it 👍